index.d.ts

```diff
@@ -0,0 +1,193 @@
+// Type definitions for neo4j-graphql-js
+// Project: https://github.com/neo4j-graphql/neo4j-graphql-js
+// Definitions by: Nopzen <https://github.com/nopzen>
+// Definitions: https://github.com/neo4j-graphql/neo4j-graphql-js.git
+// TypeScript Version: 3.8
+
+/* eslint no-unused-vars: "off" */
+
+declare module 'neo4j-graphql-js' {
+    import { Driver } from 'neo4j-driver';
+    import { GraphQLSchema, GraphQLFieldResolver, GraphQLResolveInfo, ExecutionResult, DocumentNode } from 'graphql';
+    import { IResolvers } from 'graphql-tools';
+
+    /**
+     * makeAugmentedSchema
+     * @description Wraps {@link https://www.apollographql.com/docs/apollo-server/api/apollo-server/#makeExecutableSchema|makeExecutableSchema} to create a GraphQL schema from GraphQL type definitions (SDL). Will generate Query and Mutation types for the provided type definitions and attach `neo4jgraphql` as the resolver for these queries and mutations. Either a schema or typeDefs must be provided. `resolvers` can optionally be implemented to override any of the generated Query/Mutation fields. Additional options are passed through to `makeExecutableSchema`.
+     *
+     * @param {makeAugmentedSchemaOptions} options
+     */
+    export function makeAugmentedSchema(options: makeAugmentedSchemaOptions): GraphQLSchema;
+
```

Comment:
Missing `export function assertSchema(options: any): void;`

Option types: https://grandstack.io/docs/neo4j-graphql-js-api/#parameters-1

Reply:
fixed with: 724d85f629ef27b75a24d379517aea94a38d99f

Reply:
Thank you :)